client/src/components/LoginForm.js

```diff
@@ -56,25 +60,37 @@ const LoginForm = () => {
 			</Typography>
 			<FormControl margin="dense">
 				{/* field for email */}
-				<TextField id="login-email" label="Email" type="email" margin="dense" onChange={(e) => {
-					submitObject.email = e.target.value;
-				}} />
+				<TextField
+					id="login-email"
+					label="Email"
+					type="email"
+					margin="dense"
+					autoComplete="off"
+					onChange={(e) => {
+						submitObject.email = e.target.value;
```

Comment:
Is not better to store this directly in the state submitObjectState?

Reply:
wouldn't that constantly refresh the state though? I'll give it a try see what happens :P

Reply:
yes just tried several different ways and it doesn't like it to be set whenever I make a change. If you know of a way to do this could you post syntax for me please because I can't find any working syntax for this on google